### ISSUE TITLE
npm install failing on missing dependency

### ISSUE THREAD

**Issue by fuentesjr** [josephfrazier/standard-eject#5]
Here is what I see:

```
npm install --global standard-eject
npm ERR! Error while executing:
npm ERR! /usr/local/bin/git ls-remote -h -t https://github.com/parro-it/falafel-bash.git
npm ERR! 
npm ERR! remote: Repository not found.
npm ERR! fatal: repository 'https://github.com/parro-it/falafel-bash.git/' not found
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/salfuentes/.npm/_logs/2020-10-07T15_11_27_351Z-debug.log
```

npm --version
6.14.8
node --version
v14.10.0 

Please let me know if you'd like me to provide additional information.

**Comment by josephfrazier** [comment]
Ah, it looks like we're using a particular commit in that repo, and I guess github must have garbage-collected it: https://github.com/josephfrazier/standard-eject/blob/a469fd7ebf0096f16ffd5c840b967a94f3f2da50/package.json#L24

Let me see if I can figure out why that commit is used.

**Comment by josephfrazier** [comment]
Oh, the entire repo disappeared: https://github.com/parro-it/falafel-bash

Looks like the package is on NPM though, hopefully we can use that: https://www.npmjs.com/package/falafel-bash

**Comment by josephfrazier** [comment]
I just published version 1.0.6, let me know if it's not working for you: https://www.npmjs.com/package/standard-eject/v/1.0.6

I'm also working on ~upgrading all the dependencies to get~ getting the tests passing on Travis again, will hopefully have another new version published before long.
